src/library/scanner/libraryscanner.cpp

```diff
@@ -65,6 +66,27 @@ void cleanUpDatabase(const QSqlDatabase& database) {
             << timer.elapsed().debugMillisWithUnit();
 }
 
+/// Update statistics for the query planner
+/// See also: https://www.sqlite.org/lang_analyze.html
+void analyzeAndOptimizeDatabase(const QSqlDatabase& database) {
```

Comment:
I'm a bit confused by the naming. We only run a `ANALYZE` command and do not actually enable `PRAGMA optimize` here (because that would run `ANALYZE` automatically which might be inconvenient if it happens during a set).

Wouldn't `analyzeDatabase` or `updateQueryPlannerStatistics` be a more fitting name for this function?

Reply:
OPTIMIZE doesn't work well when you are using multiple connections or a connection pool as in aoide. Please refer to the SQLite docs, the naming is confusing.

Reply:
Yes, this was just about the naming of the method.